pages/dkp/konvoy/2.1/choose-infrastructure/azure/advanced/new/index.md

```diff
@@ -53,7 +76,13 @@ enterprise: false
     > ${CLUSTER_NAME}.yaml
     ```
 
-1.  (Optional) To configure the Control Plane and Worker nodes to use an HTTP proxy:
+The output of this command is `${CLUSTER_NAME}.yaml` file that you can examine or modify. If you use this method to create a basic cluster without HTTP proxies or setting existing network infrastructure values, skip to the heading, "Inspect or edit the cluster objects."
+
+## (Optional) Configure the control plane and workers to use an HTTP Proxy
+
+This procedure uses the `--dry-run` and `--output-yaml` flags together to create basic Azure Kubernetes cluster objects in a YAML file. This approach allows you to examine the YAML objects before creating the actual Azure Kubernetes cluster itself. To configure the Control Plane and Worker nodes to use an HTTP proxy:
```

Comment:
This is a bit of 'department of redundancy department', we just said most of this in the last section:

```suggestion
To configure the Control Plane and Worker nodes to use an HTTP proxy:
```

Reply:
Resolved as noted in my local branch.